config.go

```diff
@@ -1403,6 +1415,14 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		)
 	}
 
+	// Append the network type to the sqlite directory so that it is
+	// "namespaced" per network in the same fashion as the data directory.
+	cfg.DB.Sqlite.DBPath = filepath.Join(
```

Comment:
After playing w/ things locally, I think this should instead live under the existing `chain` and `graph` subdirs. 

What did the diff before here actually look like?